Loop54.Shared/Model/Request/GetRelatedEntitiesRequest.cs

```diff
@@ -36,5 +36,16 @@ public GetRelatedEntitiesRequest(Entity entity)
         /// Parameters for specifying which results to retrieve. Such as filtering, faceting, sorting and paging. 
         /// </summary>
         public EntityCollectionParameters ResultsOptions { get; set; } = new EntityCollectionParameters();
+        
+        /// <summary>
+        /// The kind of relation that will be used to create resulting entities. Similar or Complementary.
+        /// Defaults to Similar if not specified.
+        /// </summary>
+        /// <remarks>
+        /// When getting related entities for 'BrandA bike tire': <br/>
+        /// "similar" - will return entities that are 'more of the same' (f.e. 'BrandB bike tire')<br/>
+        /// "complementary" - will return entities that are 'people also buy' (f.e. 'BrandX bike pump')
+        /// </remarks>
+        public RelationTypes? RelationKind { get; set; } = null;
```

Comment:
This is nullable to be backward-compatible with older engine versions (those throw when receiving payload with `relationKind` field).